WooCommerce/Classes/ViewModels/CardPresentPayments/PaymentCaptureCelebration.swift

```diff
@@ -0,0 +1,30 @@
+import AudioToolbox
+import UIKit
+
+/// Plays a sound and provides haptic feedback when a payment capture has been completed successfully
+final class PaymentCaptureCelebration: NSObject {
+    private var soundID: SystemSoundID = 0
+
+    func celebrate() {
+        playSound()
+        shakeDevice()
+    }
+}
+
+private extension PaymentCaptureCelebration {
+    func playSound() {
+        guard let path = Bundle.main.path(forResource: "o.caf", ofType: nil) else {
+            return
+        }
+
+        let url = URL(fileURLWithPath: path)
+        AudioServicesCreateSystemSoundID(url as CFURL, &soundID)
+        AudioServicesPlaySystemSound(soundID)
+    }
+
+    func shakeDevice() {
+        let generator = UINotificationFeedbackGenerator()
+        generator.prepare()
```

Comment:
I can't target my phone today to try this out, but does this prepare make a noticeable difference?